remo/reports/api/views.py

```diff
@@ -19,3 +30,77 @@ def retrieve(self, request, pk):
         serializer = ActivitiesDetailedSerializer(report,
                                                   context={'request': request})
         return Response(serializer.data)
+
+
+class ActivitiesKPIFilter(django_filters.FilterSet):
+    """Filter for activities KPI endpoint."""
+    category = django_filters.CharFilter(name='functional_areas__name')
+    initiative = django_filters.CharFilter(name='campaign__name')
+
+    class Meta:
+        model = NGReport
+        fields = ['country']
+
+
+class ActivitiesKPIView(APIView):
+
+    def get(self, request):
+        """Returns serialized data for Activities KPI"""
+
+        queryset = NGReport.objects.filter(report_date__lte=now())
+        activities = ActivitiesKPIFilter(request.GET, queryset=queryset)
+        weeks = int(request.query_params.get('weeks', KPI_WEEKS))
+
+        # Total calculations
+        total = activities.qs.count()
+
+        # Quarter calculations
+        current_quarter = int(math.ceil(now().month/3.0))
+        current_quarter_start = datetime(now().year, current_quarter, 1)
```

Comment:
I am not sure I can understand this. According to the var name this should be the start date of each quarter. If this is true, given June as an example month, the result will be 1st of February for the start date of the second quarter.